Library/Formula/twoping.rb

```diff
@@ -0,0 +1,18 @@
+class Twoping < Formula
+  desc "Ping utility to determine directional packet loss"
+  homepage "http://www.finnie.org/software/2ping/"
+  url "http://www.finnie.org/software/2ping/2ping-3.0.1.tar.gz"
+  sha256 "d6997cd1680151e6f7d5e60137d45cd41bf385d26029878afdaaf5dc4f63dcc4"
+
+  depends_on "Python"
+
+  def install
+    system "python", *Language::Python.setup_install_args(prefix)
+    man1.install "doc/2ping.1"
+    man1.install_symlink "2ping.1" => "2ping6.1"
+  end
+
+  test do
+    assert_equal "#{version}", shell_output("#{bin}/2ping -V 2>&1").chomp
```

Comment:
2ping is a network utility.  It includes both the client and listener (server) components, so it would be possible to set up a listener and then have the client test against it and then stop the listener, but my Ruby experience is limited and I wouldn't know how to handle the concurrency (unless you know of an existing formula which does something similar).

Alternatively, test.2ping.net is a permanent listener which the client could test against, but that means relying on an external service to test.  I'm not sure how acceptable that is w/r/t CI, policy, etc.

Another alternative is simply e.g. "2ping --listen --deadline=5", which would start up the listener, do nothing for 5 seconds and exit, which is probably better than nothing.